packages/integration-tests/index.js

```diff
@@ -536,7 +625,7 @@ async function addRulesWithPostfix(config, dataDirectory, overrides, postfix) {
         ...config
       }));
 
-      const r = new Rule();
+      const r = new RulesModel();
```

Comment:
If you're going to bother changing this line, I'd go all the way and change the single letter variable names

Reply:
Will do.